packages/localizations/src/nl-NL.ts

```diff
@@ -0,0 +1,554 @@
+        actionLabel__conectionFailed: 'Probeer opnieuw',
+        actionLabel__reauthorize: 'Authoriseer nu',
+        destructiveActionTitle: 'Verwijderen',
+        destructiveActionSubtitle: 'Verwijder dit verbonden account en verwijder het uit uw account',
+        destructiveActionAccordionSubtitle: 'Verwijder verbonden account',
+      },
+      passwordSection: {
+        title: 'Wachtwoord',
+        primaryButton__changePassword: 'Wachtwoord wijzigen',
+        primaryButton__setPassword: 'Wachtwoord instellen',
+      },
+      mfaSection: {
+        title: 'Tweestapsverificatie',
+        primaryButton: 'Tweestapsverificatie instellen',
+        phoneCode: {
+          destructiveActionTitle: 'Verwijderen',
+          destructiveActionSubtitle: 'Verwijder dit telefoonnummer van de tweestapsverificatiemethodes',
+          destructiveActionLabel: 'Verwijder telefoonnummer',
+          title__default: 'Standaardfactor',
+          title__setDefault: 'Stel in als standaardfactor',
+          subtitle__default: 'Deze factor wordt gebruikt als de standaard tweestapsverificatiemethode bij het inloggen.',
+          subtitle__setDefault:
+            'Stel deze factor in als de standaard tweestapsverificatiemethode bij het inloggen.',
+          actionLabel__setDefault: 'Stel in als standaard',
+        },
+        backupCodes: {
+          headerTitle: 'Backupcodes',
+          title__regenerate: 'Hergenereer backupcodes',
+          subtitle__regenerate:
+            'Genereer een nieuwe set backupcodes. De vorige kunnen niet meer gebruikt worden.',
+          actionLabel__regenerate: 'Hergenereer codes',
+        },
+        totp: {
+          headerTitle: 'Authenticatorapplicatie',
+          title: 'Standaardfactor',
+          subtitle: 'Deze factor wordt gebruikt als de standaard tweestapsverificatiemethode bij het inloggen.',
+          destructiveActionTitle: 'Verwijderen',
+          destructiveActionSubtitle: 'Verwijder deze authenticatorapplicatie van de tweestapsverificatiemethodes',
+          destructiveActionLabel: 'Verwijder authenticatorapplicatie',
+        },
+      },
+      activeDevicesSection: {
+        title: 'Actieve apparaten',
+        primaryButton: 'Actieve apparaten',
+        detailsTitle: 'Huidig apparaat',
+        detailsSubtitle: 'Dit is het apparaat dat u momenteel gebruikt',
+        destructiveActionTitle: 'Uitloggen',
+        destructiveActionSubtitle: 'Log uit op dit apparaat',
+        destructiveAction: 'Log uit op apparaat',
+      },
+      web3WalletsSection: {
+        title: 'Web3 portefeuilles',
+        primaryButton: 'Web3 portefeuilles',
+        destructiveActionTitle: 'Verwijderen',
+        destructiveActionSubtitle: 'Verwijder deze portefeuille van uw account',
+        destructiveAction: 'Verwijder portefeuille',
+      },
+    },
+    profilePage: {
+      title: 'Profiel bijwerken',
+      imageFormTitle: 'Profielafbeelding',
```

Comment:
`profielfoto` sounds like better alternative.

`afbeelding` is more like `depiction`.